browser/components/pages/RequestReviewPage.js

```diff
@@ -3,13 +3,16 @@ import Button from '../atoms/Button'
 import Layout from '../molecules/Layout'
 import InspectObject from '../utils/InspectObject'
 import createPrrr from '../../actions/createPrrr'
+import './RequestReviewPage.sass'
 
 export default class RequestReviewPage extends Component {
   render(){
     const { session } = this.props
-    return <Layout className="HomePage" session={session}>
+    return <Layout className="RequestPage" session={session}>
```

Comment:
thanks for catching this wrong classname. We should use `RequestReviewPage` instead of `RequestPage` so their consistently.